internal/pkg/service/stream/storage/statistics/statistics.go

```diff
@@ -40,7 +40,14 @@ type Value struct {
 
 type PerSlice struct {
 	SliceKey model.SliceKey
-	Value    Value
+
+	// The following fields are part of the Value structure, which are applicable when collecting statistics.
+
+	FirstRecordAt    utctime.UTCTime
+	LastRecordAt     utctime.UTCTime
+	RecordsCount     uint64
+	UncompressedSize datasize.ByteSize
+	CompressedSize   datasize.ByteSize
```

Comment:
To prevent misuse, only relevant fields are here, no `SlicesCount` and `StagingSize`.